contracts/event/MultipleResultsEvent.sol

```diff
@@ -535,6 +484,9 @@ contract MultipleResultsEvent is NRC223Receiver, Ownable {
         _currentResultIndex = resultIndex;
         _currentRound = _currentRound + 1;
 
+        // Clear current voting round totals
+        _currentVotingRoundTotals = [0, 0, 0, 0];
```

Comment:
I think this will trigger delete(), can you replace with a for loop to assign 0 for each?

Reply:
pretty sure assigning directly is going to be less jumps than doing a for loop.

Reply:
actually the preferred way seems to be using delete. this is the same as assigning a value = 0 to each of the slots in array. adding a for loop will add the overhead of keeping a value for the arrayIndex while looping through it.

https://solidity.readthedocs.io/en/latest/types.html#delete

Reply:
ok